index.js

```diff
@@ -96,7 +96,6 @@ const camundaCloud86Rules = withConfig({
 
 const camundaCloud87Rules = withConfig({
   ...omit(camundaCloud86Rules, [ 'no-task-listeners' ]),
-  'zeebe-user-task': 'error',
```

Comment:
I would prefer to remove the rule and all related tests for now. We don't keep rules in the codebase that aren't used.

Reply:
I think a complete rule removal is a breaking change. I don't think it's necessary for the bugfix.

Reply:
I consider the rule being there a bug. We remove it, we fix the bug. 😆